packages/core-data/src/resolvers.js

```diff
@@ -707,3 +707,14 @@ export const getNavigationFallbackId =
 			] );
 		}
 	};
+
+export const getDefaultTemplateId =
+	( query ) =>
+	async ( { dispatch } ) => {
+		const template = await apiFetch( {
+			path: addQueryArgs( '/wp/v2/templates/lookup', query ),
```

Comment:
I've discovered a different issue here while testing. If the current default template is empty the REST endpoint returns the next one available from template hierarchy. That's an issue, because front end and the editor show different things. I'll create an issue for this.